qiskit_ibm/ibm_provider.py

```diff
@@ -178,53 +152,61 @@ def __new__(
             IBMProviderCredentialsInvalidToken: If the `token` is not a valid
                 IBM Quantum token.
         """
-        account_credentials, account_preferences, hgp = cls._resolve_credentials(
+        # pylint: disable=unused-argument,unsubscriptable-object
+        super().__init__()
+        account_credentials, account_preferences = self._resolve_credentials(
             token=token,
             url=url,
-            hub=hub,
-            group=group,
-            project=project,
             **kwargs
         )
-        hub, group, project = hgp.to_tuple()
-        if not cls._credentials and (not cls._providers or
-                                     cls._is_different_account(account_credentials.token)):
-            cls._initialize_providers(credentials=account_credentials,
-                                      preferences=account_preferences)
-            return cls._get_provider(hub=hub, group=group, project=project)
-        elif not cls._credentials and cls._providers:
-            return cls._get_provider(hub=hub, group=group, project=project)
-        else:
-            return object.__new__(cls)
+        self._initialize_backend_groups(credentials=account_credentials,
+                                        preferences=account_preferences)
+        if self._backend_groups and self._backend_group:
```

Comment:
doesn't it make more sense to just raise if there are no `_backend_group[s]`? Since you can't really do anything if your account has no hgp.